api/librats_init.c

```diff
@@ -0,0 +1,118 @@
+/* Copyright (c) 2022 Intel Corporation
+ * Copyright (c) 2020-2021 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// clang-format off
+#include <stdlib.h>
+#include <string.h>
+#include <librats/log.h>
+#include <librats/err.h>
+#include "librats/attester.h"
+#include <internal/attester.h>
+#include "librats/verifier.h"
+#include <internal/verifier.h>
+#include <internal/core.h>
+
+#ifdef SGX
+#include "rats_t.h"
+#endif
+// clang-format on
+
+#ifdef SGX
+// clang-format off
+#define RATS_NUM    5
+#define RATS_NAME   32
+// clang-format on
+char rats_instance_name[RATS_NUM][RATS_NAME] = { "nullattester", "nullverifier", "sgx_ecdsa",
+						 "sgx_ecdsa_qve", "sgx_la" };
+#endif
+
+rats_err_t librats_init(rats_conf_t *conf, rats_core_context_t *ctx)
```

Comment:
`rats_core_context_t ` is an internal structure, and I think it is useless to expose it to users.

Reply:
It needs to be filled manually by the user.  It's a requirement for some users. 
For user agnostically, we can figure it out later